extensions/adapters/auth/pom.xml

```diff
@@ -0,0 +1,33 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>geowave-extension-parent</artifactId>
+        <groupId>mil.nga.giat</groupId>
+        <relativePath>../../</relativePath>
+        <version>0.9.4-SNAPSHOT</version>
+    </parent>
+    <artifactId>geowave-adapter-auth</artifactId>
+    <name>GeoWave Adapter Auth</name>
+    <description>Authorization functionality for GeoWave Data Adapters</description>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-core</artifactId>
+            <version>4.0.4.RELEASE</version>
+            <scope>provided</scope>
```

Comment:
You aren't looking at the latest commit.  Try this: https://github.com/ngageoint/geowave/pull/1066/commits/83f11a062799f26ff77518c6329c78838204ee3e#diff-c3f3a0b58ea56b7b1532923859181c72R15